tests/run/repeatable/Test_1.scala

```diff
@@ -7,16 +7,18 @@ import repeatable._
 @Plain_0(3)
 trait U
 
-@FirstLevel_0(Array(Plain_0(4), Plain_0(5)))
-@FirstLevel_0(Array(Plain_0(6), Plain_0(7)))
+@FirstLevel_0(Array(new Plain_0(4), new Plain_0(5)))
+@FirstLevel_0(Array(new Plain_0(6), new Plain_0(7)))
 trait T
 
-object Test:
-  def main(args: Array[String]) =
+object Test {
+  def main(args: Array[String]) = {
```

Comment:
> Are the changes in this file intended? 

The  addition of `new` is intended, the other ones weren't and I removed them. I've updated the commit message of the corresponding commit to explain why this was needed:

    This change is not fully backwards source compatible: this is illustrated
    by the diffs in tests/run/repeatable/Test_1.scala:

        -@FirstLevel_0(Array(Plain_0(4), Plain_0(5)))
        +@FirstLevel_0(Array(new Plain_0(4), new Plain_0(5)))

    Here, FirstLevel_0 takes an array of `Plain_0` annotations as arguments, and in
    previous releases of Scala 3 we could put `Plain_0(4)` in this array without
    `new`. This is because the compiler generates a "constructor proxy" apply method
    for classes, but this no longer works since `Plain_0` is now a trait. While we
    could potentially tweak the constructor proxy logic to handle this case, it
    seems simpler to require a `new` here, both because Scala 2 does it too and
    because it ensures that user code that inspects the annotation tree does not
    have to deal with constructor proxies.